README.md

```diff
@@ -312,3 +360,75 @@ reported properly.
 
 *NOTE*: the current implementation does not handle all edge cases at this time.
 If you send invalid data, it is possible for it to silently fail.
+
+## How to fork this add-on to create your own SudoSandwich instance on Heroku
+
+If you want to run your own add-on based on this codebase, you should follow these instructions. These instructions are for deploying your add-on to Heroku, which is not required. You may have to modify the deploy instructions for other environments.
```

Comment:
can do we do line breaks in these paragraphs for easier editing / inline commenting? my editor has an easy way to do this. if yours does not we can always do it in a separate PR.

Reply:
I usually just let me editor to the newlines for me so I usually do the first style but I can see why you might prefer the latter. Fine with either!